data/app.d/historical_data_parser.py

```diff
@@ -10,11 +10,45 @@ class MissingNullsJsonWriter:
     Attributes:
         rows (list<dict>): A list of dictionaries representing each row to write
         columns (dict): A dictionary representing the columns to write
+        datetime_converter (method): A method that takes a string and converts it
+            to a Deephaven DateTime object. If the string isn't convertable to a
+            DateTime object, None should be returned
     """
 
-    def __init__(self):
+    def __init__(self, datetime_converter=None):
```

Comment:
should default to something that parses the datetimes we see in sportradar

Reply:
The `extract_sport_radar_json` is generally how we're gonna use this anyways so that makes more sense to have the specific SportRadar default to me